dev-tools/src/main/resources/ant/integration-tests.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0"?>
+<project name="elasticsearch-integration-tests">
+
+  <!-- runs an OS script -->
+  <macrodef name="run-script">
+      <attribute name="script"/>
+      <attribute name="dir"/>
+      <attribute name="args"/>
+      <attribute name="spawn"/>
+    <sequential>
+      <exec executable="cmd" osfamily="winnt" dir="@{dir}" spawn="@{spawn}">
+        <arg value="/c"/>
+        <arg value="@{dir}/@{script}.bat"/>
+        <arg line="@{args}"/>
+      </exec>
+
+      <exec executable="sh" osfamily="unix" dir="@{dir}" spawn="@{spawn}">
+        <arg value="@{dir}/@{script}"/>
+        <arg line="@{args}"/>
+      </exec>
+    </sequential>
+  </macrodef>
+
+  <!-- unzip core release artifact and start ES -->
+  <target name="start-external-cluster">
+    <property name="integ.finalname" value="${project.artifactId}-${project.version}"/>
+    <property name="integ.scratch" location="${project.build.directory}/integ-tests"/>
+
+    <unzip src="${project.build.directory}/releases/${integ.finalname}.zip"
+           dest="${integ.scratch}"/>
+
+    <property name="integ.home" location="${integ.scratch}/${integ.finalname}"/>
+    <property name="integ.repo.home" location="${integ.home}/repo"/>
+    <property name="integ.args"
+              value="-Des.node.name=smoke_tester -Des.cluster.name=prepare_release
+                     -Des.discovery.zen.ping.multicast.enabled=false -Des.script.inline=on
+                     -Des.script.indexed=on -Des.path.repo=${integ.repo.home}"/>
+
+    <!-- execute -->
+    <echo>Starting up external cluster...</echo>
+    <run-script dir="${integ.home}" script="bin/elasticsearch" args="${integ.args}" spawn="true"/>
+
+    <waitfor maxwait="3" maxwaitunit="minute" checkevery="500">
+      <http url="http://127.0.0.1:9200"/>
```

Comment:
Would it be possible to make this configurable or at least move it to some other (non-default) port?